src/js/dataMapping/keyword/resultsTableColumns.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * resultsTableColumns.js
+ * Created by Lizzie Salita 1/5/18
+ */
+
+const contractColumns = [
+    'Award ID',
+    'Mod',
```

Comment:
`Mod` and `Awarding Sub Agency` are causing HTTP 400 errors: `Elasticsearch error`
These fields do not appear to be in the [gist](https://github.com/fedspendingtransparency/usaspending-api/wiki#possible-contract-fields-w-es-mapping)